### ISSUE TITLE
DropdownItem Link seems not to work

### ISSUE THREAD

**Issue by endeffects** [chanan/BlazorStrap#95]
I tried to use your Navbar Component today and used it as follow,
but the href link of the DropdownItem is not called when i click on it.

```
<Navbar IsExpand="true">
    <NavbarBrand href="#">Brand</NavbarBrand>
    <NavbarToggler onclick="@(async () => await OnClick())" />
    <Collapse isOpen="@IsOpen" IsNavbar="true">
        <BlazorNav IsList="true" IsNavbar="true">
            <NavItem>
                <BlazorNavLink IsActive="true" href="#" Match="NavLinkMatch.All">Home</BlazorNavLink>
            </NavItem>
            <NavItem IsDropdown="true">
                <DropdownToggle>CategoryOne</DropdownToggle>
                <DropdownMenu>
                    <DropdownItem href="CategoryOne/Create">Create</DropdownItem>
                    <DropdownItem href="CategoryOne/Delete">Delete</DropdownItem>
                    <DropdownItem href="CategoryOne/Rename">Rename</DropdownItem>
                    <DropdownItem href="CategoryOne/Show">Show</DropdownItem>
                </DropdownMenu>
            </NavItem>
            <NavItem IsDropdown="true">
                <DropdownToggle>CategoryTwo</DropdownToggle>
                <DropdownMenu>
                    <DropdownItem href="CategoryTwo/Create">Create</DropdownItem>
                    <DropdownItem href="CategoryTwo/Delete">Delete</DropdownItem>
                    <DropdownItem href="CategoryTwo/Rename">Rename</DropdownItem>
                    <DropdownItem href="CategoryTwo/Show">Show</DropdownItem>
                </DropdownMenu>
            </NavItem>
        </BlazorNav>
    </Collapse>
</Navbar>
```

**Comment by chanan** [comment]
I will look into it tonight or tomorrow

**Comment by chanan** [comment]
This bug happens because of the onlostfocus of NavItem, looking into how to fix it

**Comment by chanan** [comment]
Fixed. Will be released in v1.0.0-preview6-20

**Comment by endeffects** [comment]
Thank you. :)

**Comment by endeffects** [comment]
@chanan The problem still exists im version 20.

**Comment by chanan** [comment]
I will check it out

**Comment by chanan** [comment]
I cant reproduce the broken link anymore. Are you using CSB or SSB?

I uploaded a the sample that works for me at: https://github.com/chanan/BlazorStrap/blob/master/src/SampleCore/Pages/Samples/Issue95.razor

**Comment by endeffects** [comment]
I'm using Blazor Client Side. I've noticed that it sometimes works and if this is the case, the DropdownMenu stays opened. With the BlazorNavLink itself i don't have any problems.

**Comment by jbomhold3** [comment]
``` html
<DropdownMenu> = ul
<DropdownItem>Create</DropdownItem> = li  li's do not have an href. the 
````
correct format should be
``` html
<DropdownItem">
<BlazorNavLink href="CategoryOne/Create" Match="NavLinkMatch.All">Crete</BlazorNavLink></DropdownItem>
````

**Comment by endeffects** [comment]
Oh ok, i'll try this. I used your sample from https://chanan.github.io/BlazorStrap/navbars

![image](https://user-images.githubusercontent.com/13432569/61072019-b8e6a280-a412-11e9-94bd-64f21c6b0b35.png)

**Comment by jbomhold3** [comment]
stand by a moment.  just double checked the class

**Comment by chanan** [comment]
Second attempt for a fix in 1.0.0-preview6-21 (Not ready to go out yet, we are putting a few more fixes in for other issues)
